airtime_mvc/application/models/Preference.php

```diff
@@ -498,7 +498,12 @@ public static function SetStationDescription($description)
 
     public static function GetStationDescription()
     {
-        return self::getValue("description");
+        $description = self::getValue("description");
+        if (!empty($description)) {
+            return $description;
+        } else {
+            return "Powered by Airtime Pro";
```

Comment:
From a localization point of view I would suggest translating the whole string, because in some languages the word order might be: 'Airtime Pro powered by' even if you do not translate 'Airtime Pro' itself. You might also want the flexibility to insert another brand name here for a white-label client, by changing only the .po file.